common/src/main/java/com/viaversion/viabackwards/api/rewriters/LegacyBlockItemRewriter.java

```diff
@@ -93,7 +93,7 @@ private void addMappings(MappedLegacyBlockItem.Type type, JsonObject object, Int
     private void addMapping(String key, JsonObject object, MappedLegacyBlockItem.Type type, Int2ObjectMap<MappedLegacyBlockItem> mappings) {
         int id = object.getAsJsonPrimitive("id").getAsInt();
         JsonPrimitive jsonData = object.getAsJsonPrimitive("data");
-        short data = jsonData != null ? jsonData.getAsShort() : 0;
+        short data = jsonData != null ? jsonData.getAsShort() : -1;
```

Comment:
I think you need to change the default vb mappings to explicitly set 0 then, but the better alternative would probably be to revert this line and set it to -1 in vr mappings